### ISSUE TITLE
Connect sub-synth and add-synth's cutoff and resonance to existing zyn controls

### ISSUE THREAD

**Issue by musikBear** [LMMS/lmms#711]
This is an old 'problem'. When the lmms GUI was made, the lmms' FREQ and RES was connected to zasfx C.freq and Q resp. in the zasfx engine. A pecularity in zasufx means that the Q connection only is active if the chosen filter is a pole2. the two pole1 (LP/HPf1) has no handle. If possible these should also be handled from lmms-gui. Both the two other synths (sub-synt and PADsynth also needs handles)
All of this is perhaps futile to tacle, if a complete new version of zasfx are to be included in 1.1, then i gues these omitions should be disregarded. However -when- the new implemettion zasfx is made, these connetions should definately be made (and if for some reason the 'new' zasfx is prosponed, then the handles should be fixed in a ..next? release :)

**Comment by Umcaruje** [comment]
@musikBear can this request be concidered a duplicate of #1965?

**Comment by musikBear** [comment]
no, not really. ENV/LFO tab just gives room for more direct zasfx dials 
This topic is about connecting sub-synth and add-synth's resonance and frequency to the existing dials (front-tab)
BUT
there is no reason to try to do this for **ADD-synth**, and it may even be causing serious problems if it was done!
ADD-synth's need for actually building a complete new table of samples for *every* change in settings, would create havoc, it a dial was automated
Or the short version:
Any attempt to automate anything in ADD-synth will create problems!

Sub-synth CO & reso and BW however, should be connected to the existing front-tab dials (imo)

**Comment by Umcaruje** [comment]
Ok, it makes sense now. Tagging @curlymorphic for his opinion, since he is doing the zyn 2.5 upgrade.

**Comment by musikBear** [comment]
:+1: 
@Umcaruje ...sorry to have to ask, but what was that thing you asked for video about
I cant find that notification any longer..
Can i somehow get old already read notifications back? :speak_no_evil:

**Comment by curlymorphic** [comment]
This feature may soon be solved.

ZASF has a midi learn feature, That I did test, and added some extra knobs to the lmms zyn instument host, These then could be assigned to contols in zasf, The issue at the time, was zasf never saved these "midi learns", so the contols and recorded automation would no longer work after the current session.
 I have spoken to @fundamental about this, with a view to future versions saving this informaton.

**Comment by Umcaruje** [comment]
>Can i somehow get old already read notifications back? :speak_no_evil:

Yeah you can, just go to the [notifications page](https://github.com/notifications) and select 'All notifications':

![image](https://cloud.githubusercontent.com/assets/6282045/8429048/59da8c60-1f24-11e5-9221-5238904439a7.png)

**Comment by musikBear** [comment]
:p ahh you 'caught me' before i edited the post, i found it just after i have posted the first time :p 
Its my eyes, they are not doing so well with gray text on white background, so i diddent see the sidebar
Thanks for bothering to help me :+1:
